lib/offsite_payments/integrations/bit_pay.rb

```diff
@@ -23,12 +23,19 @@ class Helper < OffsitePayments::Helper
         def initialize(order_id, account, options)
           super
           @account = account
-
           add_field('posData', {'orderId' => order_id}.to_json)
           add_field('fullNotifications', true)
           add_field('transactionSpeed', 'high')
         end
 
+        if options[:checkout_token].length >=44
```

Comment:
You can reduce this to:

if options[:checkout_token].length >=44 && /^[^0OIl]*$/.match(options[:checkout_token])
    self.invoicing_url = 'https://bitpay.com/invoices'
end

Reply:
thanks , I updated the PR.

@pi3r  this should be a cleaner implementation and switch the API endpoint based on 
the token length and if certain characters are present that would only exist in the older version

Reply:
Hey Joshua, thanks for this. Could you add a test case to cover this change? Also this seems to be causing the Bitpay unit tests to fail.